packages/integrations/react/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@astrojs/react",
   "description": "Use React components within Astro",
-  "version": "4.0.0",
+  "version": "4.1.0",
```

Comment:
There's no need to update the version. That will be done automatically by changesets. You can revert the change